### PR TITLE
Mention `dart pub add`

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

File: src/_guides/packages.md [hunk 1]
```diff
@@ -54,6 +54,18 @@ dependencies:
   intl: ^0.17.0
 ```
 
+Rather than editing the pubspec manually, 
+you can also add a dependency by running a `dart` tool command. 
+This adds a dependency on `vector_math`:
```
Comment (atsansone) [inline src/_guides/packages.md:59]: **issue:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
To update the `pubspec.yaml` file, you have a better option than hand editing. 
To add a dependency, you can run `dart pub add` command.
The following example adds a dependency on `vector_math`.
```

**Review by atsansone — CHANGES_REQUESTED** [review]
@mit-mit : Need to rephrase to meet Google Style.

File: src/_guides/packages.md (hunk 1 above)
Comment (mit-mit) [reply src/_guides/packages.md]: Thanks! Committed a variant of that (wanted to avoid "hand" editing)

**Review by atsansone — APPROVED** [review]
LGTM!

**Pull request merged**